### DISCUSSION TITLE
WebFX: same name?

### DISCUSSION THREAD

**Discussion by brunoborges** [webfx-project/webfx#9] [Q&A]
I appreciate the initiative here. It's really awesome to see such project come to life. But as the creator of WebFX in JavaFX (github.com/brunoborges/webfx), it would be nice if the authors had approached for a dialogue regarding using the same name.

**Comment by salmonb** [comment]
Hi Bruno,

Apologies for the long reply delay, the project has been paused for nearly a year, but is now restarting.

Right, same name, sorry about that... I started the project a while ago, and called it webfx at that time, like many other small projects on GitHub (there are more than 50 repositories called webfx). I had no idea how far it will go, and it's only last year that its popularity started to increase.

It would be quite difficult to rename it now. And even so, I can't really see a better name, as WebFX describes exactly what it is (a web port of JavaFX). Also it's short enough to use it for our CLI, which is convenient for people to just type `webfx` commands in the terminal.

I'm sorry! Let me know if you are still concerned about that name.
